patches/0005-LPS-112982-Add-a-languageId-parameter-to-resource-UR.patch

```diff
@@ -0,0 +1,45 @@
+From 5b126a54b6e53f23345ebbcfc9741c5a48a25e31 Mon Sep 17 00:00:00 2001
+From: Julien Castelain <julien.castelain@liferay.com>
+Date: Mon, 6 Jul 2020 09:39:27 +0200
+Subject: [PATCH 5/5] LPS-112982 Add a "languageId" parameter to resource URLs
+
+---
+ core/ckeditor_base.js | 20 +++++++++++++-------
+ 1 file changed, 13 insertions(+), 7 deletions(-)
+
+diff --git a/core/ckeditor_base.js b/core/ckeditor_base.js
+index 3fb18f5f7..8e6f33018 100644
+--- a/core/ckeditor_base.js
++++ b/core/ckeditor_base.js
+@@ -171,15 +171,21 @@ if ( !window.CKEDITOR ) {
+ 			 * @returns {String} The full URL.
+ 			 */
+ 			getUrl: function( resource ) {
+-				// If this is not a full or absolute path.
+-				if ( resource.indexOf( ':/' ) == -1 && resource.indexOf( '/' ) !== 0 )
+-					resource = this.basePath + resource;
++				var resourceUrl = resource.includes(CKEDITOR.basePath)
```

Comment:
Would this be clearer as `startsWith(CKEDITOR.basePath)`? (I don't know; I'm assuming it is an absolute URL.)